database/migrations/2020_10_20_092958_change_default_migration.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class ChangeDefaultMigration extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('internet_accesses', function (Blueprint $table) {
+            $table->integer('auto_approved_mac_slots')->default(1)->change();
```

Comment:
Don't we want to change the existing ones also?

Reply:
I've checked manually.

Reply:
Ah, now I realised your question. Yes, we want that... will create a PR.